frontend/components/panels/NotifCheckBox.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of Search NEU and is licensed under AGPL3.
+ * See the license file in the root folder for details.
+ */
+
+import React from 'react';
+import { Checkbox } from 'semantic-ui-react';
+import user from '../user';
+import macros from '../macros';
+import Keys from '../../../common/Keys';
+
+// This file renders the checkboxes that control which sections a user signs up for
+// notifications.
+
+export default class NotifCheckBox extends Checkbox {
```

Comment:
Extend React.component - like all the other components - and then just use checkbox in the render method